modules/olm-updating-index-image.adoc

```diff
@@ -35,20 +35,43 @@ endif::[]
 ----
 $ opm index add \
     --bundles <registry>/<namespace>/<new_bundle_image>@sha256:<digest> \//<1>
-    --from-index <registry>/<namespace>/<existing_index_image>:<tag> \//<2>
-    --tag <registry>/<namespace>/<existing_index_image>:<tag> \//<3>
-    --pull-tool podman <4>
+    --from-index <registry>/<namespace>/<existing_index_image>:<existing_tag> \//<2>
+    --tag <registry>/<namespace>/<existing_index_image>:<updated_tag> \//<3>
+    --pull-tool podman //<4>
+----
+<1> The `--bundles` flag specifies a comma-separated list of additional bundle images to add to the index.
+<2> The `--from-index` flag specifies the existing index that was previously pushed.
```

Comment:
This is fine, but if you want to avoid passive, could say:
```suggestion
<2> The `--from-index` flag specifies the existing, previously pushed index.
```